src/Assert.php

```diff
@@ -897,6 +897,11 @@ protected static function strlen($value)
         return mb_strwidth($value, $encoding);
     }
 
+    protected static function createException($message)
```

Comment:
Yeah, you're right! I did not think about this possibility and will fix the name.